delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/s3/client/package.scala

```diff
@@ -0,0 +1,22 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.s3
+
+import cats.MonadThrow
+import cats.effect.IO
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.collection.NonEmpty
+import eu.timepit.refined.refineV
+import cats.syntax.all._
+import fs2.Stream
+import fs2.aws.s3.models.Models.{BucketName, FileKey}
+
+package object client {
+
+  type StreamIO[A] = Stream[IO, A]
+
+  def parseNonEmptyString[F[_]: MonadThrow](s: String): F[String Refined NonEmpty] =
```

Comment:
There is `NonEmptyString.unsafeFrom` no ?

Reply:
Yeah I just prefer to avoid unsafe methods when unnecessary. Removed now as we do that elsewhere 👍